docs/README.md

```diff
@@ -27,6 +27,14 @@ This adapter can read/write the following fields:
 - Velocity (read + write)
 - Velocity gradient (read + write)
 
+In addition, the adapter supports the following fields in a volume coupling setup:
+
+- Temperature (read* + write)
+- Pressure (read* + write)
+- Velocity (read + write)
+
+(*) Technical foundation available, the user needs to provide additional, application-specific code for enforcing the source terms.
+
```

Comment:
I polished a bit the documentation and added volume coupling in the overview page of the adapter documentation.

@tirgendetwas is this formulation correct?

Reply:
I'm not sure, what do you mean by "the user needs to provide additional, application-specific code"? We haven't managed to read Pressure so far. Temperature should probably work if it's done through the enthalpy, but we haven't tried that

Reply:
For temperature, I understand that one just needs to provided a codedSource that corresponds to their thermophysical model, no?

I can remove the reading part from temperature and pressure.